hwcomposer/hwcomposer.pro

```diff
@@ -28,7 +28,7 @@ SOURCES += hwcomposer_backend_v11.cpp
 HEADERS += hwcomposer_backend_v11.h
 
 
-QT += core-private compositor-private gui-private platformsupport-private
```

Comment:
So good question.
The dependency to compositor-private is already set into the code source of qt5-qpa-hwcomposer-plugin.
So I changed it to waylandcompositor-private to compile a version for 5.7.0 support. compositor-private was provided by QtWayland but it is now renamed waylandcompositor-private.

So I removed the waylandcompositor-private/compositor-private from the pro file and I compiled the code in a Qt 5.7.0 environment without issue.

(As mentioned previously I can't test the plugin on my device but I have a similar code for my qt5-qpa-surfaceflinger-plugin and it's working fine with the change requested).

I have already amended the commit (Qt 5.7.0 support).

Unfortunately I can't quickly do the same test with a Qt 5.6.0 environment.